CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Bump `joni` from 2.1.44 to 2.1.45
 - Bump `commons-io:commons-io` from 2.7 to 2.11.0
 - Bump `org.jruby.joni:joni` from 2.1.45 to 2.1.48
+- Bump `snakeyaml` from 1.33 to 2.0 ([#6511](https://github.com/opensearch-project/OpenSearch/pull/6511))
```

Comment:
Do you plan to backport this change to 2.x?

Reply:
Yes, please do backport (and move this to the 2.x section in this PR prior to the backport).

Reply:
Pushed the change!